LoggerApp.java

```diff
@@ -1 +1,26 @@
+package com.company;
 
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.XMLFormatter;
+
+public class LoggerApp {
+
+    public static void main(String[] args) {
+        configureLogging();
+    }
+
+    private static void configureLogging() {
+        final Logger logClassA = Logger.getLogger("org.stepic.java.logging.ClassA");
```

Comment:
why did you make this variable final?

Reply:
Because i saw many examples in the Internet that we should make it final.
Does it work correct or no?

Reply:
for this task better use simple variables, without final